test/js/bun/shell/bunshell.test.ts

```diff
@@ -661,6 +661,24 @@ describe("deno_task", () => {
     await TestBuilder.command`echo 1 > $EMPTY`.stderr("bun: ambiguous redirect: at `echo`\n").exitCode(1).run();
 
     await TestBuilder.command`echo foo bar > file.txt; cat < file.txt`.ensureTempDir().stdout("foo bar\n").run();
+
+    await TestBuilder.command`BUN_DEBUG_QUIET_LOGS=1 ${BUN} -e ${"console.log('Stdout'); console.error('Stderr')"} 2>&1`
```

Comment:
you can remove BUN_DEBUG_QUIET_LOGS from these, it should do that automatically now